tests/test_aperture.py

```diff
@@ -143,6 +143,11 @@ def test_luvoir_a_lyot_stop():
 
 	check_aperture_against_reference(make_luvoir_a_lyot_stop, 'luvoir_a_lyot', 15, options)
 
+def test_luvoir_b_aperture():
+	check_against_reference(make_luvoir_b_aperture(), 8.0, 'luvoir_b/pupil')
+	check_aperture_against_reference(make_luvoir_b_aperture, 'luvoir_b', 8, {'normalized': [(False, ''), (True, '_normalized')]})
```

Comment:
Do you want me to also split it out into a `test_luvoir_b_aperture_all()` function, like in the LUVOIR A case?

Reply:
That split was made for time reasons (that's why there is a `@pytest.mark.slow` there). LUVOIR B has less segments, plus no spiders, so that probably is a lot quicker (less permutations + less work per item). So it's likely not necessary to split them.

Reply:
Understood.